packages/protocol/contracts/bridge/BridgeCustomErrors.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+//  _____     _ _         _         _
+// |_   _|_ _(_) |_____  | |   __ _| |__ ___
+//   | |/ _` | | / / _ \ | |__/ _` | '_ (_-<
+//   |_|\__,_|_|_\_\___/ |____\__,_|_.__/__/
+
+pragma solidity ^0.8.18;
+
+abstract contract BridgeCustomErrors {
+    // Libraries having custom errors in place
```

Comment:
I believe the reason why we want to define a `BridgeCustomErrors` is that we want all custom errors which may be encountered when calling `Bridge` contract to be a part of the `Bridge` contract's compiled ABI, which will be helpful for the clients that only import the `Bridge` ABI to parse those errors. so i think we should also add those custom errors defined in `LibBridgeXXX` to here?

Reply:
Make sense and done.